errbot/backends/irc.py

```diff
@@ -440,6 +473,30 @@ def on_dcc_connect(self, dcc, event):
     def on_dcc_disconnect(self, dcc, event):
         self.transfers.pop(dcc)
 
+    def on_part(self, connection, event):
+        """
+            Handler of the part IRC Message/event.
+
+            The part message is sent to the client as a confirmantion of a
+            /PART command sent by someone in the room/channel.
+            If the event.source contains the bot nickname then we need to fire
+            the :meth:bot.callback_room_left event on the bot.
+
+            :param:
+                connection: Is an 'irc.client.ServerConnection' object
+
+            :param:
+                event: Is an 'irc.client.Event' object
+                The event.source contains the nickmask of the user that
+                leave the room
+                The event.target contains the channel name
```

Comment:
This param syntax is wrong, these should go on the same line, as in `:param event:` otherwise Sphinx will fail to parse it.